bin/projects/dbatools/dbatools.Tests/dbatools.Tests.csproj

```diff
@@ -34,9 +34,9 @@
   </PropertyGroup>
   <Import Project="$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets" Condition="Exists('$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets')" />
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.0-preview-20220726-02" />
-    <PackageReference Include="MSTest.TestAdapter" Version="2.3.0-preview-20220810-02" />
-    <PackageReference Include="MSTest.TestFramework" Version="2.3.0-preview-20220810-02" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
```

Comment:
as discussed, while it totally makese sense to use pre-release of all the SQL related nuget packages that get released slowly, the tst stuff gets released at a normal cadance and we are not doing bleeding edge unit tests here.